tests/ui/perfherder/alerts-view/alerts_test.jsx

```diff
@@ -251,6 +251,28 @@ describe('alert filtering ignores repository and/or options', () => {
   });
 });
 
+test('Debug Tools heading not displayed for non-browsertime summaries', async () => {
+  const alertSummary1 = testAlertSummaries[0];
```

Comment:
Instead of adding to the `tests/ui/mock/alert_summaries.json` you might be able to take the existing mocks and just modify the fields that you are interested in (eg alertSummary.framework_id = 13)

Reply:
This is what I did, but if I replace one of the summaries the other tests will fail. The `Debug Tools` column is for browsertime, and the first summary in the mocks is talos. If I change that to browsertime other tests will fail.